protocol/messenger.go

```diff
@@ -84,6 +85,10 @@ const (
 	privateChat ChatContext = "private-chat"
 )
 
+// Set this to true to test the feature in development.
+// TODO remove this when the feature is hooked to the UI
+const newsFeedFeatureFlag = false
```

Comment:
We have feature flags here:
https://github.com/status-im/status-go/blob/ad28f15531c76c216f08118d4055a140ee52f80f/protocol/common/feature_flags.go#L3